src/helm/benchmark/scenarios/scenario.py

```diff
@@ -49,39 +49,39 @@ def unpack_tag(tag: str) -> Tuple[str, str]:
     return key, value
 
 
-class Input(ABC):
+@dataclass(frozen=True)
+class Input:
     """
-    The text corresponding to the input of an Instance. We want to subclass this for structure inputs (e.g., QA).
+    The input of an `Instance`.
     """
 
-    @abstractmethod
-    def to_text(self):
-        pass
+    text: str
 
 
 @dataclass(frozen=True)
-class RawInput(Input):
+class PassageQuestionInput(Input):
     """
-    Contains a single text string.
+    Passage-question pair used for question answering Scenarios.
     """
 
-    text: str
-
-    def to_text(self):
-        return self.text
+    def __init__(
+        self,
+        passage: str,
+        question: str,
+        passage_prefix: str = "",
+        question_prefix: str = "Question: ",
+        separator: str = "\n",
+    ):
+        super().__init__(f"{passage_prefix}{passage}{separator}{question_prefix}{question}")
```

Comment:
Idle musing: if the only thing that downstream code accesses from this class is `text`, then this class might as well be just a `TextInput`...

Reply:
Actually, I think we want to represent `PassageQuestionInput` as a dataclass....technically, the prefix/separator should be part of the adapter, but it will be hard to weave this information in since it's dependent on the class type here...maybe we should use promptsource?  Anyway, we can save this for another PR.